templates/project/Makefile.twig

```diff
@@ -30,6 +30,18 @@ lint-xml:
 
 .PHONY: lint-xml
 
+lint-xliff:
```

Comment:
You can add this to the generic `lint` task.

Reply:
Done